src/services/Utils.ts

```diff
@@ -18,10 +18,10 @@ import rundef from 'rundef';
 
 export class Utils {
     static range = (min: number, max: number): number[] => {
-        const len = max - min - 1;
+        const len = max - min + 1;
```

Comment:
so this makes the length inclusive. when displaying range(0, 9), the len is 10. previously the gensis block was always ignored and the range(0, 9) had a len of 8.

Reply:
if only the genesis block is available on a testnet for example range(0, 0) now produces (as it should) [0].